Client/Clique/Common/Models/Participants.swift

```diff
@@ -0,0 +1,10 @@
+//
+//  Participants.swift
+//  Clique
+//
+//  Created by Infinum on 13.12.2022..
+//
+
+import Foundation
+
+struct Participant
```

Comment:
I don't think this will compile, did you check?

Reply:
It compiles. Not sure why it wouldn't work tho?

Reply:
Another question is more suitable - where in the Xcode do you see this file?

Reply:
It would not work because it should be defined properly. It should be defined as:

```swift
struct Participant {
}
```

But this class is already defined, so it will not compile even after you define it properly.

In short, that file should be removed, but currently it is removed only from the Xcode - file still exists.

Reply:
I'll leave a comment here; right-click on _Clique > Common > Models_ and select _Show in Finder_. You'll see that you have a file `Participants.swift` which is not included into the Xcode project. You can delete that file from the file system.